pkg/profile/artifact_test.go

```diff
@@ -27,12 +27,16 @@ import (
 const (
 	subscriptionName     = "mySub"
 	namespace            = "default"
-	branch               = "main"
+	branch               = "support-helm-urls"
```

Comment:
TODO: Don't forget to change this to main once https://github.com/weaveworks/nginx-profile/pull/6 is merged.